user-interface/src/main/java/life/qbic/datamanager/views/projects/create/CollaboratorsLayout.java

```diff
@@ -20,121 +24,101 @@
  * within a project during project creation and validates the provided information</p>
  */
 @Component
-public class CollaboratorsLayout extends Div implements HasValidation {
+public class CollaboratorsLayout extends Div implements HasBinderValidation<ProjectCollaborators> {
 
-  private static final String TITLE = "Project Collaborators";
-  private final ContactField principalInvestigatorField = new ContactField(
-      "Principal Investigator");
-  private final ContactField responsiblePersonField = new ContactField(
-      "Project Responsible/Co-Investigator (optional)");
-  private final ContactField projectManagerField = new ContactField("Project Manager");
-  private final Binder<ProjectCollaborators> collaboratorsBinder = new Binder<>(
-      ProjectCollaborators.class);
+  private final AutocompleteContactField principalInvestigatorField;
+  private final ContactField responsiblePersonField;
```

Comment:
Is there a reason why the responsiblePersonField is not an AutoCompleteContactField?